usr/bin/steam-de

```diff
@@ -93,7 +93,8 @@ if settings["on_deck"] in (True, "on", "yes"):
 if settings["deck_ui_mode"] in (True, "on", "yes"):
     _COMMANDS["start_steam"].append("-gamepadui")
     _COMMANDS["start_steam"].append("-steampal")
-
+else
+    _COMMANDS["start_steam"].append("-oldbigpicture")
```

Comment:
Since the new BPM is taking over, instead of adding this `else` statement, I feel it would be better if we replaced the entire `if`/`else` block with this:
```python3
if settings["deck_ui_mode"] in (False, "off", "no", "disable"):
    _COMMANDS["start_steam"].append("-oldbigpicture")
```

Reply:
I thought maybe there will be people with old version trying to launch Steam in deck mode but you are right: Steam auto-updates itself sooner or later.